vscode/src/completions/inline-completion-item-provider.ts

```diff
@@ -695,3 +699,78 @@ function onlyCompletionWidgetSelectionChanged(prev: CompletionRequest, next: Com
 
     return prevSelectedCompletionInfo.text !== nextSelectedCompletionInfo.text
 }
+
+/**
+ * Handles showing an in-editor decoration when a first completion is accepted.
+ */
+class FirstCompletionDecorationHandler {
+    /**
+     * Duration to show decoration before automatically hiding.
+     *
+     * Modifying the document will also immediately hide.
+     */
+    private static readonly decorationDurationMilliseconds = 3000
+
+    /**
+     * A subscription watching for file changes to automatically hide the decoration.
+     *
+     * This subscription will be cancelled once the decoration is hidden (for any reason).
+     */
+    private editorChangeSubscription: vscode.Disposable | undefined
+
+    /**
+     * A timer to hide the decoration automatically.
+     */
+    private hideTimer: NodeJS.Timeout | undefined
+
+    private readonly decorationType = vscode.window.createTextEditorDecorationType({
+        after: {
+            margin: '0 0 0 40px',
+            contentText: '    🎉 Accepted your "first" Cody autocomplete!',
```

Comment:
After playing with it locally, let's change the message to:

```suggestion
            contentText: '    🎉 You just accepted your first Cody autocomplete!',
 ```